src/main/kotlin/graphics/scenery/FullscreenObject.kt

```diff
@@ -9,6 +9,9 @@ import graphics.scenery.attribute.material.Material
  * @author Ulrik Günther <hello@ulrik.is>
  */
 class FullscreenObject : Mesh("FullscreenObject") {
+
+    var wantsSync = true
+    override fun wantsSync(): Boolean = wantsSync
```

Comment:
wantsSync() comes from the interface and has a default implementation of "=true"  which is what you want in most cases. If you want to be able to change this at runtime, you have to do the above. 

(Judging from the name of the class wantsSync will never be changed and those lines could be removed.)

Reply:
Thanks @moreApi for the explanation. 72ef4eebeb3389722609611f084b50270c4ba8d6 changes the default value of `wantsSync` to `false` as that is more appropriate for this class. I've left the lines of code in there as it may make sense in some cases for this to be changeable at runtime